src/index.ts

```diff
@@ -49,12 +49,19 @@ import { Components } from "./components.js"
 import { Configuration, namespace } from "./configuration.js"
 import { isString, Maybe } from "./common/index.js"
 import { Session } from "./session.js"
-import { loadFileSystem } from "./filesystem.js"
+import { loadFileSystem, recoverFileSystem } from "./filesystem.js"
 import FileSystem from "./fs/filesystem.js"
 
 
+// TYPES
+
+
+import { type RecoverFileSystemParams } from "./fs/types/params"
+
+
 // IMPLEMENTATIONS
 
+
```

Comment:
adjusted the spacing here to be consistent with the other import/export sections at the top of the file